Comp2022/src/main/include/commands/Climber3ROTATETOL3.h

```diff
@@ -0,0 +1,41 @@
+// RobotBuilder Version: 3.1
+//
+// This file was generated by RobotBuilder. It contains sections of
+// code that are automatically generated and assigned by robotbuilder.
+// These sections will be updated in the future when you export to
+// C++ from RobotBuilder. Do not put any code or make any change in
+// the blocks indicating autogenerated code or it will be lost on an
+// update. Deleting the comments indicating the section will prevent
+// it from being updated in the future.
+
+#pragma once
+
+// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=INCLUDES
+
+#include "subsystems/Climber.h"
+
+#include <frc2/command/CommandBase.h>
+#include <frc2/command/CommandHelper.h>
+
+// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=INCLUDES
+#include <frc2/command/SequentialCommandGroup.h>
+
+using namespace std;
+
+/**
+ *
+ *

```

Comment:
Can you please fill out these comment sections with a summary what the step is meant to do? This and all the other `.h` files

Reply:
The summaries are in the .cpp files, but I can move them into the .h file for better readability if needed

Reply:
Oops I didn't realize - if that's the case then can you delete the comment headers in the `.h` files? Seeing a block like this makes me think there _should_ be some words, but if the block wasn't there then I would know to look elsewhere